docs/migration/manual-migration.mdx

```diff
@@ -824,4 +842,4 @@ In v3, the sitemap URL has been changed from `/sitemaps/sitemap.xml` to
 - The `Sitemapable` GraphQL interface has been deprecated for removal. It can be
   removed from your project.
 - The `SitemapImage` GraphQL type has been deprecated for removal. It can be
-  removed from your project.
+  removed from your project. import {info} from "autoprefixer"
```

Comment:
**issue:** not needed

**suggestion:** remove

Reply:
I have no idea how it got there 🤔 

Removed.